src/common/NetworkConditions.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2021 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { NetworkConditions } from './NetworkManager.js';
+
+export type NetworkConditionsMap = { [name: string]: NetworkConditions };
+
+export const networkConditionsMap: NetworkConditionsMap = {
```

Comment:
Shouldn't the list cover all the options from the list `"none"| "cellular2g"| "cellular3g"| "cellular4g"| "bluetooth"| "ethernet"| "wifi"| "wimax"` maybe omitting `other`?

Reply:
Please take a look at my other comment in regards to the relationship between the list here and the `connectionType`.